Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -115,7 +115,7 @@
       <Version>6.1.0-build.6</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.3.200213001</Version>
+      <Version>2.4.0</Version>
```

Comment:
Since we are doing this now, lets use latest, 2.4.2, shipped yesterday hehe.

Reply:
Ah, ok, didn't realize they shipped minor updates, good to know! I'll update and re-test.